independent-publisher-connectors/Pushover/apiDefinition.swagger.json

```diff
@@ -0,0 +1,361 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Pushover",
+    "description": "Pushover makes it easy to get real-time notifications on your Android, iPhone, iPad, and Desktop.",
+    "contact": {
+      "name": "Glen Hutson",
+      "email": "msconnectors@glenhutson.com"
+    },
+    "version": "1.0"
+  },
+  "host": "api.pushover.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/1/messages.json": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@glenhutson May I ask you please to use more proper description. you can use "OK", "Success", "Bad Request", "200", etc. Please fix all responses descriptions in the swagger file.

Reply:
@Amjed-Ayoub , descriptions updated.